tests/platform_tests/api/test_sfp.py

```diff
@@ -423,3 +423,33 @@ def test_power_override(self, duthost, localhost, platform_api_conn):
             if self.expect(power_override is not None, "Unable to retrieve transceiver {} power override data".format(i)):
                 self.expect(power_override is False, "Transceiver {} power override data is incorrect".format(i))
         self.assert_expectations()
+
+    def test_get_position_in_parent(self, platform_api_conn):
+        for i in range(self.num_sfps):
+            position = sfp.get_position_in_parent(platform_api_conn, i)
+            self.expect(position is not None, "Failed to perform get_position_in_parent for sfp {}".format(i))
+            self.expect(isinstance(position, int), "Position value must be an integer value for sfp {}".format(i))
+        self.assert_expectations()
+
+    def test_is_replaceable(self, platform_api_conn):
+        for i in range(self.num_sfps):
+            replaceable = sfp.is_replaceable(platform_api_conn, i)
+            self.expect(replaceable is not None, "Failed to perform is_replaceable for sfp {}".format(i))
+            self.expect(isinstance(replaceable, bool), "Replaceable value must be a bool value for sfp [}".format(i))
```

Comment:
The second `self.expect()` call is irrelevant if the first fails, so it should be nested using `if self.expect()` on the first. See lines 416-417 as an example.